test/test.py

```diff
@@ -1189,6 +1189,115 @@ def testSearchReturnFields(self):
             self.assertEqual('doc:1', total[0].id)
             self.assertEqual('telmatosaurus', total[0].txt)
 
+    def test_text_params(self):
+        conn = self.redis()
+
+        with conn as r:
+            # Creating a client with a given index name
+            client = Client('idx', port=conn.port)
+            client.redis.flushdb()
+            client.create_index((TextField('name'),))
+
+            client.add_document('doc1', name='Alice')
+            client.add_document('doc2', name='Bob')
+            client.add_document('doc3', name='Carol')
+
+            params_dict = {"name1":"Alice", "name2":"Bob"}
+            q = Query("@name:($name1 | $name2 )").set_params_dict(params=params_dict)
```

Comment:
I prefer to have something like:
```
params = {"name1":"Alice", "name2":"Bob"}
q = Query("@name:($name1 | $name2 )")
res1 = client.search(params=params)
res2 = client.search(params=other_params)
```
WDYT?

Reply:
we have it, continue reading the test ;)